chapters/bash-tools.Rmd

```diff
@@ -90,7 +90,7 @@ $ cat lengths.txt
 We can now use `sort` to sort the lines in this file:
 
 ```bash
-$ sort lengths.txt
+$ sort lengths.txt -n
```

Comment:
Can't replicate the results in the next code block without the option. Following lines of prose may need to be updated.

Reply:
If you update the sentence following this command so that it says "we use `sort`'s `-n` option" instead of "we should use `sort`'s `-n` option" then that should be the only update the prose needs.